src/ECS/utilities/SparseArray.hpp

```diff
@@ -8,28 +8,41 @@
 #ifndef SPARSE_ARRAY_HPP
```

Comment:
The header contains a wrong note, its RType not BSRtype (it's my bad, but it could be time to change it !)

Reply:
It's been resolved as well